client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -88,14 +89,19 @@ class ProductCreate extends React.Component {
 	}
 
 	render() {
-		const { siteId, product, className, variations, productCategories } = this.props;
+		const { siteId, product, className, variations, productCategories, actionList } = this.props;
+
+		const isReady = 'undefined' !== siteId;
```

Comment:
I don't think this is working as intended. I am able to click Save (and get an error) when the sites/site is not-yet-loaded:

![error-when-saving-before-loaded](https://user-images.githubusercontent.com/1595739/27235403-927914b6-5275-11e7-9494-0e21bf5c8540.gif)